home/.chezmoiscripts/universal/run_onchange_after_57-elastic-agent.sh.tmpl

```diff
@@ -0,0 +1,29 @@
+{{- if and (ne .host.distro.family "windows") (stat (joinPath .host.home ".config" "age" "chezmoi.txt")) (or (stat (joinPath .chezmoi.sourceDir ".chezmoitemplates" "secrets" "ELASTIC_TOKEN")) (env "ELASTIC_TOKEN")) -}}
+#!/usr/bin/env bash
+# @file Netdata
```

Comment:
This should be renamed to Elastic Fleet Server

Reply:
Done.

Reply:
This doesn't look done..

Reply:
This is still @file Netdata